apps/nextjs-website/src/lib/webinars/__tests__/webinarQuestions.test.ts

```diff
@@ -26,6 +31,7 @@ const makeTestWebinarEnv = () => {
   // default mock behaviour
   dynamoDBClientMock.send.mockImplementation((cmd) => {
     if (cmd instanceof PutItemCommand) return Promise.resolve({});
+    else if (cmd instanceof UpdateItemCommand) return Promise.resolve({});
     else if (cmd instanceof QueryCommand)
       return Promise.resolve({ Items: [aDynamoDBItem] });
     // eslint-disable-next-line functional/no-promise-reject
```

Comment:
It's more concise and readable to use an async function instead of explicitly rejecting or resolving promises.

```suggestion
  dynamoDBClientMock.send.mockImplementation(async (cmd) => {
    if (cmd instanceof PutItemCommand) return {};
    else if (cmd instanceof UpdateItemCommand) return {};
    else if (cmd instanceof QueryCommand)
      return { Items: [aDynamoDBItem] };
    else throw new Error('Unsupported command')
```